site/man/rabbitmq-streams.8.html

```diff
@@ -303,7 +314,155 @@ <h3 class="Ss" title="Ss" id="Stream_plugin"><a class="permalink" href="#Stream_
+      <dd>The name of the super stream to delete.</dd>
+      <dt><var class="Ar" title="Ar">vhost</var></dt>
+      <dd>The virtual host of the super stream.
+        <div class="Pp"></div>
+        Delete a super stream.</dd>
+    </dl>
+    <div class="Pp"></div>
+  </dd>
+  <dt><a class="permalink" href="#list_stream_consumer_groups"><code class="Cm" title="Cm" id="list_stream_consumer_groups">list_stream_consumer_groups</code></a>
+    [<div class="Op"><code class="Fl" title="Fl">-p</code>
+    <var class="Ar" title="Ar">vhost</var></div>]
+    [<div class="Op"><var class="Ar" title="Ar">groupinfoitem
+    ...</var></div>]</dt>
+  <dd>
+    <div class="Pp"></div>
+    Lists groups of stream single active consumers for a vhost.
+    <div class="Pp"></div>
+    The <var class="Ar" title="Ar">groupinfoitem</var> parameter is used to
+      indicate which group information items to include in the results. The
+      column order in the results will match the order of the parameters.
+      <var class="Ar" title="Ar">groupinfoitem</var> can take any value from the
+      list that follows:
+    <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">consumers</var></dt>
+      <dd>Number of consumers in the group.</dd>
+      <dt><var class="Ar" title="Ar">partition_index</var></dt>
+      <dd>The stream partition index if the stream is part of a super stream, -1
+          if it is not.</dd>
+      <dt><var class="Ar" title="Ar">reference</var></dt>
+      <dd>The group reference (name).</dd>
+      <dt><var class="Ar" title="Ar">stream</var></dt>
+      <dd>The stream the consumers are attached to.</dd>
+    </dl>
+    <div class="Pp"></div>
+    If no <var class="Ar" title="Ar">groupinfoitem</var> are specified then
+      stream, reference, partition_index, and consumers are displayed.
+    <div class="Pp"></div>
+    For example, this command displays the stream, reference, and number of
+      consumers for each group:
+    <div class="Pp"></div>
+    <div class="Bd Bd-indent lang-bash"><code class="Li">rabbitmq-streams
+      list_stream_consumer_groups stream reference consumers</code></div>
+    <div class="Pp"></div>
+  </dd>
+  <dt><a class="permalink" href="#list_stream_group_consumers"><code class="Cm" title="Cm" id="list_stream_group_consumers">list_stream_group_consumers</code></a>
+    <code class="Fl" title="Fl">--stream</code>
+    <var class="Ar" title="Ar">stream</var>
+    <code class="Fl" title="Fl">--reference</code>
+    <var class="Ar" title="Ar">reference</var>
+    [<div class="Op"><code class="Fl" title="Fl">--vhost</code>
+    <var class="Ar" title="Ar">vhost</var></div>]
+    [<div class="Op"><var class="Ar" title="Ar">consumerinfoitem
+    ...</var></div>]</dt>
+  <dd>
+    <div class="Pp"></div>
+    Lists consumers of a stream consumer group in a vhost.</dd>
+  <dt><var class="Ar" title="Ar">stream</var></dt>
+  <dd>The stream the consumers are attached to.</dd>
```

Comment:
I would change this: "The stream the consumers are attached to" to this:

"The stream to which consumers are attached. "